lib/Ed25519Signature2020.js

```diff
@@ -107,19 +110,27 @@ export class Ed25519Signature2020 extends LinkedDataSignature {
   }
 
   async assertVerificationMethod({verificationMethod}) {
+    let contextUrl;
+    let requiredKeyType;
+    if(verificationMethod.type === 'Ed25519VerificationKey2018') {
+      contextUrl = SUITE_CONTEXT_URL_2018;
+      requiredKeyType = 'Ed25519VerificationKey2018';
+    } else if(verificationMethod.type === 'Ed25519VerificationKey2020') {
+      contextUrl = SUITE_CONTEXT_URL;
+      requiredKeyType = 'Ed25519VerificationKey2020';
+    }
```

Comment:
Looks like strange errors would be thrown if the case where neither type is found here and `contextUrl` remains `undefined`.  A final `else` that throws an unsupported type error would be good.

Reply:
Added here fce8215